docs/platform/custom-cloud-credentials/troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+# Bring Your Own Cluster (BYOC) Troubleshooting
+
+## Deploying BYOC
+
+### Kubernetes API Authentication Errors
+AWS STS regional endpoints have to be enabled in the target region. Go to https://console.aws.amazon.com/iam/home#/account_settings and verify the regional endpoint is activated. In most cases your region already has AWS STS regional endpoint enabled -> https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html
```

Comment:
1. What happens if endpoints are not enabled? Could you rephrase this to be in this format: In order to do [a thing], AWS STS regional endpoints have to be enabled in the target region. 

2. Create a text hyperlink instead of just pasting the link. So you can update that line to 
Go to your [Amazon account settings](https://console.aws.amazon.com/iam/home#/account_settings) and verify...

3. add: "that" (...and verify **that** the regional endpoint is activated).

Reply:
4. In most cases your region already has AWS STS regional endpoint enabled -> 
Can you specify in a hyper-linked sentence what this link is pointing to?